src/resources/js/components/AccountEdition.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="container">
+    <h2>My Account</h2>
+
+    <div v-if="showMessage" class="error">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <form @submit.prevent="submitChange()">
+      <label for="UpdateFirstName">
```

Comment:
Add an `id` to the input it's bound to.

`the for attribute specifies which form element a label is bound to` 

[Source](https://www.w3schools.com/TAGS/att_for.asp)